examples/kubernetes-hello-world/kubernetes-hello-world.md

```diff
@@ -0,0 +1,35 @@
+# Kubernetes
```

Comment:
Should we have the title as `Kubernetes - Hello World`

Reply:
Fixed.